apps/web/src/state/transactions/updater.tsx

```diff
@@ -50,10 +49,7 @@ export const Updater: React.FC<{ chainId: number }> = ({ chainId }) => {
       (transaction) => {
         const getTransaction = async () => {
           try {
-            const receipt = await waitForTransaction({
-              hash: transaction.hash,
-              chainId,
-            })
+            const receipt: any = await provider.waitForTransactionReceipt({ hash: transaction.hash })
```

Comment:
Correct me if I'm wrong. So changing to use the `waitForTransactionReceipt` from provider is to prevent the wagmi wrapper from throwing error so we can toast error [here](https://github.com/pancakeswap/pancake-frontend/pull/7136/files#diff-7cf13ed80cc566159f67b4de7e56c226970328f377b6835e715068b9117aaac7L75)?